swiftgen.yml

```diff
@@ -0,0 +1,63 @@
+xcassets:
```

Comment:
You should follow a no-dead code policy. So, you may need to delete any unused / commented code.